app/src/main/java/ar/com/wolox/android/login/LoginPresenter.kt

```diff
@@ -7,10 +7,7 @@ import javax.inject.Inject
 
 class LoginPresenter @Inject constructor(private val userSession: UserSession) : BasePresenter<LoginView>() {
 
-    private fun isEmailValid(email: String): Boolean {
-
-        return android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
-    }
+    private fun isEmailValid(email: String): Boolean = android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
```

Comment:
I missed this line in the previous PR. You can create a String extension function for this
https://kotlinlang.org/docs/extensions.html